locales/en/translations.json

```diff
@@ -1,4 +1,18 @@
 {
     "startTitle" : "This is Limited Partnerships",
-    "errorStartAddressMissing" : "Please enter an address"
+    "errorStartAddressMissing" : "Please enter an address",
+
+    "pageNotFound_Title" : "Page not found",
```

Comment:
it would be better to have sections (nested objects) rather than prefixes

"pageNotFound": {
   ....
}

Reply:
Ah yes! That will be much better. Will check this works and implement.

Reply:
Not working but @sbasile-ch is working on a fix so will test and hopefully address this soon.

Reply:
Ok. Working now, with a new release of the `ch-node-utils` lib. Changed.